### PR TITLE
refactor: concat forwardProps option

### PR DESCRIPTION
**Pull request by miguelc1221**
#63

**What**: concat forwardProps option metadata

**Why**:  Necessary in order to not have to pass in the same props again. See #63.

**How**: Created a function that returns a ternary in order to avoid the eslint complexity rule. Used the function on the `forwardProps` metadata in order to return correct forwardsProps.

### CODE REVIEW

**Comment by miguelc1221** [comment]
I added unit but don't have much experience writing tests. Let me know if there is anything i should fix = ). Thanks

File: src/__tests__/index.test.js [hunk 1]
```diff
@@ -105,6 +105,15 @@ test('merges composed component styles for reasonable overrides', () => {
   expect(wrapper).toMatchSnapshotWithGlamor()
 })
 
+test('merges composed component forwardProps', () => {
+  const parent = ({shouldRender, ...rest}) =>
+    (shouldRender ? <div {...rest} /> : null)
+  const Child = glamorous(parent)({forwardProps: ['shouldRender']})
```
Comment (kentcdodds) [inline src/__tests__/index.test.js]: This should be:

```javascript
glamorous(parent, {forwardProps: ['shouldRender']})()
```

**Comment by kentcdodds** [comment]
Thanks a ton for this work by the way. I'm excited to get this in!

File: src/__tests__/index.test.js (hunk 1 above)
Comment (miguelc1221) [inline src/__tests__/index.test.js]: ah you're right, fixed.

**Pull request merged**